tests/vmi_multus_test.go

```diff
@@ -215,7 +215,7 @@ var _ = Describe("Multus", func() {
 					&expect.BExp{R: "\\$ "},
 					&expect.BSnd{S: cmdCheck},
 					&expect.BExp{R: "\\$ "},
-					&expect.BSnd{S: "ip addr show eth1 | grep 10.1.1 | wc -l"},
+					&expect.BSnd{S: "ip addr show eth1 | grep 10.1.1 | wc -l\n"},
 					&expect.BExp{R: "1"},
```

Comment:
this is also another reason i dont think we should be depend on correct previous expecters, in one of the threads here, i can find it if needed

Reply:
Did you missed this one?

Reply:
@EdDev yes, we don't wait for the prompt and we should, relevant here and relevant in may other places. The PR doesn't aim to fix all the places.

Reply:
aint it wait because it use the new mechanism which does wait ?
this is why i overlooked it

it even doesnt wait for \n1 so it must use the new mech unless it already does. (which what i glanced and believed is does)

Reply:
This commit claims to fix this test. As you touched it, I would expect you to leave it in a good shape.

Reply:
btw it made me think, since CheckForTextExpecter uses now ExpectBatchWithValidatedSend
which wait for the prompt on each Exp that use RetValue
we arent allowed to wait for prompt after use of RetValue in case there is (didnt find so far) or will be one in the future.